vars/database.yml

```diff
@@ -0,0 +1,38 @@
+---
+database_host: localhost
+database_port: 5432
+database_ssl: false
+database_ssl_mode: disable
+database_ssl_ca: None
+
+foreman_database_name: foreman
+foreman_database_user: foreman
+foreman_database_password: CHANGEME
+candlepin_database_user: candlepin
+candlepin_database_name: candlepin
+candlepin_database_password: CHANGEME
+pulp_database_user: pulp
+pulp_database_name: pulp
+pulp_database_password: CHANGEME
+
+candlepin_db_user: "{{ candlepin_database_user }}"
+candlepin_db_name: "{{ candlepin_database_name }}"
+candlepin_db_password: "{{ candlepin_database_password }}"
+candlepin_db_host: "{{ database_host }}"
+candlepin_db_port: "{{ database_port }}"
+candlepin_db_ssl_mode: "{{ database_ssl_mode }}"
+candlepin_db_ssl_ca: "{{ database_ssl_ca }}"
+
+pulp_settings_db_env:
```

Comment:
@evgeni Am I doing this right here? Or do you expect each of the entries to be their own variable mapped to a secret that creates an environment variable?